tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature

```diff
@@ -1,4 +1,6 @@
-@issue-ocis-1317
+@issue-ocis-1317 @skipOnOCIS
+# this scenario is skipped on ocis because it opens share folder which in not possible in OCIS
+# but it works for OC10 see issue https://github.com/owncloud/web/issues/6896 for more detail
```

Comment:
I put comment at the top because all scenario was skipped because of the same reason

Reply:
You could add issue tag to https://github.com/owncloud/web/issues/6896

Reply:
@saw-jan  (https://github.com/owncloud/web/issues/6896) and  (https://github.com/owncloud/web/issues/1317) sre inter-related so its better to put only one issue tag

Reply:
I don't think they are interrelated. https://github.com/owncloud/web/issues/6896 is about share-jail and https://github.com/owncloud/ocis/issues/1317 is about ability to set various admin settings (like `occ` in oC10)

Reply:
sorry I mean adding both issue links and tags overloads the feature with information.

Reply:
Yeah, indeed we should not cluster scenario with tags but since both issues are different and both related to the scenario, it would be good to add them both, specially #6896